client.go

```diff
@@ -164,27 +165,30 @@ func NewClient(config *Config, remote string) (*Client, error) {
 			info.RemoteConfig.Addr = fmt.Sprintf("unix:%s", shared.VarPath("unix.socket"))
 		}
 	} else {
-		// Read the client certificate (if it exists)
-		clientCertPath := path.Join(config.ConfigDir, "client.crt")
-		if shared.PathExists(clientCertPath) {
-			certBytes, err := ioutil.ReadFile(clientCertPath)
-			if err != nil {
-				return nil, err
-			}
 
-			info.ClientPEMCert = string(certBytes)
+		certf := config.ConfigPath("client.crt")
+		keyf := config.ConfigPath("client.key")
+
+		if !shared.PathExists(certf) || !shared.PathExists(keyf) {
+			fmt.Fprintf(os.Stderr, i18n.G("Generating a client certificate. This may take a minute...")+"\n")
```

Comment:
Given @tych0's [comment](https://github.com/lxc/lxd/pull/2075/#discussion_r65605901) in #2075, it seems like this message belongs somewhere else.  It also raises the question of whether or not NewClient() should be generating the certificate in the first place.  After rereading the current code, I think there's an implication that it's acceptable to call NewClient() without valid certificates.